dev/tests/proxy.rs

```diff
@@ -110,6 +110,12 @@ impl Proxy {
             .resolve(&hostname, self.bind_address)
             .build()?;
 
+        let path = if let Some(path) = path.strip_prefix("/") {
+            path
+        } else {
+            path
+        };
+
```

Comment:
I don't think I understand this, why does it return path in both branches?

Reply:
In the first branch, `path` is shadowed by the `if let Some(path)` assignment. So it's the same variable name in both branches, but they refer to a different value. I probably could have used `unwrap_or` here instead, I'll clean that up next time I see it.